### PR TITLE
Add Sample Code for API-Pagepropnames

### PR DESCRIPTION
**Pull request by aashuli**

### CODE REVIEW

**Comment by aashuli** [comment]
@srish I have added code sample for API:Pagepropnames.
Here is the link of created documentaion page for API:Pagepropnames(https://www.mediawiki.org/wiki/User:Aashuli/Sandbox/API:Pagepropnames)
Kindly give your reviews so that I can make the required changes.
Would love your feedback
Thank you!

File: README.md [hunk 1]
```diff
@@ -84,8 +84,6 @@ Code snippets in Python demonstrating how to use various modules of the [MediaWi
   * [watch.py](python/watch.py): add a page to your watchlist 
 * [API:Alllinks](https://www.mediawiki.org/wiki/API:Alllinks)
   * [get_alllinks.py](python/get_alllinks.py): list links to a namespace
-* [API:Exturlusage](https://www.mediawiki.org/wiki/API:Exturlusage)
```
Comment (srish) [inline README.md]: Put this file back in the README.

File: README.md [hunk 2]
```diff
@@ -94,10 +92,9 @@ Code snippets in Python demonstrating how to use various modules of the [MediaWi
   *  [set_page_language.py](python/set_page_language.py): change page language
 * [API:Embeddedin](https://www.mediawiki.org/wiki/API:Embeddedin)
   * [get_embedded_pages.py](python/get_embedded_pages.py): get all page(s) that embed a page
-* [API:Rollback](https://www.mediawiki.org/wiki/API:Rollback)
```
Comment (srish) [inline README.md]: Same for this, put these files back.

File: README.md [hunk 3]
```diff
@@ -94,10 +92,9 @@ Code snippets in Python demonstrating how to use various modules of the [MediaWi
...
-  * [rollback.py](python/rollback.py): rollback the last edits made to a given page
-* [API:Allfileusages](https://www.mediawiki.org/wiki/API:Allfileusages)
-  * [get_allfileusages.py](python/get_allfileusages.py): list of all file usages
+* [API:Pagepropnames](https://www.mediawiki.org/wiki/API:Pagepropnames)
+  *  [get_pagepropnames.py](python/pagepropnames.py): List page property names on the given wiki
```
Comment (srish) [inline README.md]: Change the description to `List all page property names in use on the wiki.`

File: javascript/get_pagepropnames.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+//This file is autogenerated. See modules.json and autogenerator.py for details
```
Comment (srish) [inline javascript/get_pagepropnames.js]: Remove this JS file as this pull request is only about adding Python file :)

File: modules.json [hunk 5]
```diff
@@ -382,19 +382,7 @@
 			"ucuser": "Jimbo Wales"
 		}
 	},
-  	{
```
Comment (srish) [inline modules.json]: Don't remove this module.

File: modules.json [hunk 6]
```diff
@@ -382,19 +382,7 @@
...
-	   "filename": "get_allfileusages",
```
Comment (srish) [inline modules.json]: Put this module back in too.

File: modules.json [hunk 7]
```diff
@@ -405,28 +393,15 @@
             "eititle": "Computer",
             "eilimit": "20"
         }
-   	},
```
Comment (srish) [inline modules.json]: Put this module back in.

File: modules.json [hunk 8]
```diff
@@ -405,28 +393,15 @@
...
-        "docstring": "Demo of `Usercontribs` module: List user contributions.",
-        "endpoint": "https://en.wikipedia.org/w/api.php",
-        "params": {
-            "action": "query",
-        	"format": "json",
-            "list": "usercontribs",
-            "ucuser": "Jimbo Wales"
-    	}
     },
-	{
```
Comment (srish) [inline modules.json]: It looks like `get_exturlusage` module details were removed too in this pull request.

File: javascript/get_pagepropnames.js (hunk 4 above)
Comment (srish) [reply javascript/get_pagepropnames.js]: Also remove `python/__pycache__/delete.cpython-37.pyc`.

**Comment by aashuli** [comment]
Thank you @srish  for the feedback, I will make all the required changes and make a new pull request.

**Comment by srish** [comment]
@aashuli It looks like you've opened another pull request around this topic. Ideally, you submit requested changes to the same pull request. Might be something to consider for next time..

**Pull request closed without merging**